frontend/src/components/teamsAndOrgs/messages.js

```diff
@@ -237,6 +237,14 @@ export default defineMessages({
     id: 'management.organisations.stats.cost.estimation',
     defaultMessage: 'Estimated cost by the end of {year}',
   },
+  estimatedTierCostLabel: {
+    id: 'management.organisations.stats.cost.estimation.label',
+    defaultMessage: '{cost} {discountLabel}',
```

Comment:
It's better to handle it on the code instead of generating a string for translation that doesn't need translation.